.github/workflows/javascript_publish.yml

```
@@ -87,6 +86,7 @@ jobs:
         run: |
           echo "### NPM Packages" > release-notes.txt
           echo "- [@aragon/sdk-${{ needs.check_tag.outputs.package }} version ${{ needs.check_tag.outputs.version }}](https://www.npmjs.com/package/@aragon/sdk-${{ needs.check_tag.outputs.package }})" > release-notes.txt
+          echo " " > release-notes.txt
```

This will  just wipe the text file

```suggestion
          echo "### NPM Packages" > release-notes.txt
          echo "- [@aragon/sdk-${{ needs.check_tag.outputs.package }} version ${{ needs.check_tag.outputs.version }}](https://www.npmjs.com/package/@aragon/sdk-${{ needs.check_tag.outputs.package }})" >> release-notes.txt
          echo " " >> release-notes.txt
```

---

added extra `>` to all pipes